src/match/validators.py

```diff
@@ -0,0 +1,32 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+
+def validate_times_list(value):
+    updated_list = []
+    for value_element in value:
+        try:
+            nested_list = value_element.strip().split(",")
+            for index in range(len(nested_list)):
+                if nested_list[index] != "":
+                    nested_list[index] = float(nested_list[index])
+            updated_list.append(str(nested_list)[1:-1])
+        except (ValueError, TypeError):
+            raise ValidationError(
+                _(f"{value} is not a valid list of times"),
+                params={"value": value},
+            )
+    value = updated_list
+
+
+def validate_int_list(value):
```

Comment:
wait are you even validating or are you converting the list to an int list 

im not sure why this is needed, this is better: https://stackoverflow.com/questions/6009589/how-to-test-if-every-item-in-a-list-of-type-int

Reply:
^ rekt, you can do it in a single line

also @alanna-zhou i think this is him trying to create an int list and if it fails for some reason then that means there is something that is not an int in the list, meaning the list is bad... but i'm not sure why we even need to validate this because this is being referenced in the `UpdateMatchController` which should only be called internally to obscure away the details of the implementation from the frontend so they only need to call a route and we take care of the rest

Reply:
yeah if this is just to validate frontend's request i'd just say forget about it. it wastes precious time

Reply:
the reason I have this is so that frontend sends us the correctly-formatted lists when sending `proposed_locations` and `proposed_meeting_times` but since you guys think we don't need to validate those I'll just take this out